### ISSUE TITLE
python-wapiti to conda-forge

### ISSUE THREAD

**Issue by adbeda** [adsva/python-wapiti#20]
hi @adsva 

I work with tools in the open source community through the [Anaconda package manager]() under the [conda-forge channel](https://anaconda.org/anaconda/repo) and I recently run into [libwapiti](https://anaconda.org/conda-forge/libwapiti), which I'm suspecting that the way it was added has no direct link to the work you have here on github. It seems that another person packaged and distributed the repo on [pypi.org](https://pypi.org/project/libwapiti) but there is no information about the license under which it was released. 

If you're interested in sharing your work to the Anaconda community, I would like to correct the errors others made by linking the package directly to your github account as well as with your permission and licensing. On your end, this w'd require adding a [license type](nsource.org/licenses/alphabetical) of your choice and creating a [release version](https://github.com/adsva/python-wapiti/releases) of your work on github. 

Thank you for contributing the wrapper and, hopefully, people will at least start to show some level appreciation when they depend on your work.

Regards,
-Eddie

**Comment by agcopenhaver** [comment]
@adsva any movement on this? Would be very useful, as this package could be used as an upstream dependency for many packages in conda-forge.

Thanks!

**Comment by adsva** [comment]
@adbeda Sorry, missed your message for some reason. This is a pretty old repo, and I'm not sure everything still works as it should, no Python 3 support, etc. But in case anyone still finds it useful, I have followed your instructions and added a license and a release, is that enough to help you woth the packaging situation?

**Comment by agcopenhaver** [comment]
@adsva would you be able to do the same for https://github.com/adsva/libwapiti

**Comment by adsva** [comment]
@agcopenhaver I suppose, but I think the separate libwapiti repo is a bit outdated, since there have been changes in the version bundles in this repo. Perhaps the cleaner solution is to update the old repo, remove the libwapiti directory here and add it as a submodule instead, just like cwapiti. Would you agree?

**Comment by agcopenhaver** [comment]
Yeah, that probably makes more sense based on the age of libwapiti
